components/Storyblok/SbStoryCard.tsx

```diff
@@ -38,18 +39,20 @@ export const SbStoryCard = ({
     storyPicker: {
       content: {
         title = '',
-        cardTitle = '',
         dek = '',
         topics = [],
         heroImage: { filename: heroFilename = '', focus: heroFocus = '' } = {},
         bgImage: { filename: bgFilename = '', focus: bgFocus = '' } = {},
-        mobileImage: { filename: mobileFilename = '', focus: mobileFocus = '' } = {},
```

Comment:
Got rid of the mobileImage field because we don't need it anymore (at least for MVP)